lib/apiservers/templates/server/server.gotmpl

```diff
@@ -1,225 +0,0 @@
-package {{ .APIPackage }}
```

Comment:
we needed this for 030764afb6f7475524844bd94c9cdca2edf5510d . How do we handle it with new swagger?